cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
When it comes to capturing errors from promises I've found no better way than to use the ES7 `await` syntax with the regenerator runtime. However, as Levi suggests, if all errors in the promise chain are reported to `catch` I don't see a good reason to change. (Not to mention the support of `await..async` is still kinda sketch in a Babel stack from what I've experienced thus far.